airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/JsonSchemaType.java

```diff
@@ -11,6 +11,19 @@
 import java.util.Map;
 import java.util.Objects;
 
+/**
```

Comment:
_HUGE_ `+1` to more documentation like this clarifying the functionality and where the new protocol values are located